tests/builder/features_test.py

```diff
@@ -675,3 +675,22 @@ def test_comments_in_classes(ufo_module):
             ];
 """
     )
+
+
+def test_disabled_prefixes(ufo_module):
+    # https://github.com/googlefonts/glyphsLib/issues/562
+    font = to_glyphs([ufo_module.Font()])
+    feature = classes.GSFeaturePrefix(name="disabledTest")
+    feature.code = "Not even feature code"
+    feature.disabled = True
+    font.featurePrefixes.append(feature)
+
+    (ufo,) = to_ufos(font, ufo_module=ufo_module)
+    assert ufo.features.text == dedent(
+        """\
+        # Prefix: disabledTest
+        # disabled
+        # Not even feature code
+
+    """
+    )
```

Comment:
Could you check that when going back to Glyphs.app the commented-out feature prefix comes back uncommented? = in the same situation before going to UFO (if that sounds like a reasonable thing to have)

Reply:
@simoncozens ping